server/src/models/message.model.ts

```diff
@@ -0,0 +1,65 @@
+import mongoose, { Schema, Types } from 'mongoose';
+
+interface IMessageModel {
+	user_id: Types.ObjectId;
+	text: string;
+	room_id: Types.ObjectId;
+	read_by: Types.ObjectId[];
```

Comment:
i felt like read_by should be an array because we might want to say which user in the group read the message